README.md

```diff
@@ -97,13 +110,38 @@ make a tarball.
 
 To run it, create a file named `example.yaml` and run:
 
-    debos example.yaml
+    $ debos example.yaml
 
 The final tarball will be named "debian.tgz" if you would like to modify
 this name, you can provided a different name for the variable image like
 this:
 
-    debos -t image:"debian-arm64.tgz" example.yaml
+    $ debos -t image:"debian-arm64.tgz" example.yaml
+
+## Run example (using Docker)
+
+```$ sudo chmod 777 /dev/kvm```
```

Comment:
for debian at least; you should add user into kvm group and reboot

Reply:
Strictly speaking a reboot is not required - using `newgrp` is usually sufficient. Or if you have the luxury, logout / login should do the trick.

Reply:
for debian kvm is tagged with uaccess so it will be accessible to the logged in user so the kvm group isn't necessarily needed.. Generally documentation should not suggest chowning device files, it may make sense to have a seperate note on how to check/get access to /dev/kvm but it is distro/situation specific.